src/test/java/net/sf/jabref/logic/layout/format/WrapFileLinksTest.java

```diff
@@ -0,0 +1,92 @@
+package net.sf.jabref.logic.layout.format;
+
+import java.util.Collections;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+
+public class WrapFileLinksTest {
+
+    private WrapFileLinks formatter;
```

Comment:
What does this class do? The class name is not intuitive to me, maybe a candidate for a rename?!